src/modules/collection/utils.ts

```diff
@@ -135,3 +136,14 @@ export function isTPCollection(collection: Collection): boolean {
 export function getCollectionFactoryContract(chainId: ChainId) {
   return getContract(ContractName.CollectionFactoryV3, chainId)
 }
+
+export const getTPThresholdToReview = (totalItems: number) => {
+  // Reference: https://governance.decentraland.org/proposal/?id=f69c4d40-aaaf-11ec-87a7-6d2a41508231
+  // Max to review, 300 items, min to review is 50 or 1% of the collection
+  if (totalItems! < MIN_TP_ITEMS_TO_REVIEW) {
```

Comment:
Does it have a `!` because it used to be undefined?
```suggestion
  if (totalItems < MIN_TP_ITEMS_TO_REVIEW) {
```